### PR TITLE
Swerve Constants, Calibration File -> master

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

**Comment by Sh-Pe** [comment]
#527 reopened

**Review by Nitay4 — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		);
+
+		this.rotationDegreesPIDController.enableContinuousInput(-MathConstants.HALF_CIRCLE.getDegrees(), MathConstants.HALF_CIRCLE.getDegrees());
+	}
+
+	static final Rotation2d WHEEL_RADIUS_CALIBRATION_VELOCITY_PER_SECOND = Rotation2d.fromRotations(0.5);
+
+	public static final double AIM_ASSIST_MAGNITUDE_FACTOR = 4;
+
+	static final double DRIVE_NEUTRAL_DEADBAND = 0.05;
```
Comment (hillelv) [inline src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java:47]: units?

File: src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	static final Rotation2d ROTATION_NEUTRAL_DEADBAND = Rotation2d.fromRadians(0.05);
```
[the remark this answers is not in the document]
Comment (YHerm) [reply src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java:48]: its not for module, its for the rotational velocity of the swerve

[the remark this answers is not in the document]
Comment (hillelv) [reply src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java:48]: then consider heading

**Review by hillelv — APPROVED** [review]

**Review by Nitay4 — APPROVED** [review]

**Review by yonnbro — DISMISSED** [review]

**Pull request merged**